pkg/s3/s3.go

```diff
@@ -48,8 +48,16 @@ func GetConfig() *aws.Config {
 		region = "us-east-1"
 	}
 
+	accessKeyID := os.Getenv("S3_ACCESS_KEY_ID")
+	secretAccessKey := os.Getenv("S3_SECRET_ACCESS_KEY")
+
+	creds := credentials.NewStaticCredentials(accessKeyID, secretAccessKey, "")
```

Comment:
why not check if the accessKey and the secretKey are set first? (like the other two changes)
for example
```go
var creds *credentials.Credentials
if accessKeyID != "" || secretAccessKey != "" {
	creds = credentials.NewStaticCredentials(accessKeyID, secretAccessKey, "")
}
```